tests/test_serializers/test_json.py

```diff
@@ -208,6 +210,77 @@ def test_decode_collection_error():
             JsonDeserializer.decode_collection(value)
 
 
+class TestPathEncoding:
+    """ Tests for custom encoding of Path types. """
+
+    PATH_DECODING_TEST_CASES = [
+        (pathlib.Path(), CustomEncodedPathTypes.path.json_encoding([])),
```

Comment:
Could you make all these test paths non-trivial? It would be safer and would make it easier for new eyes to understand what's going on